vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_fields.html.haml

```diff
@@ -26,3 +26,6 @@
   .row
     .span8
       = f.submit text("shared.search"), class: "btn"
+      - if params[:return_path].present?
+        = link_to t("shared.cancel"), params[:return_path]
```

Comment:
I feel like we could improve the layout of this view a little bit. I think the compose mail button shouldn't go further right than the table's right edge. There probably doesn't need to be as much of a gap between the two buttons. And "Cancel" could probably be in a better spot.

![screen shot 2016-10-04 at 3 39 36 pm](https://cloud.githubusercontent.com/assets/1099111/19091571/d9b7c32c-8a48-11e6-8365-73e3a98a8eb4.png)